src/components/ProductsBlock/Sidebar/Checkbox/index.tsx

```diff
@@ -9,28 +10,30 @@ interface PropTypes {
   id?: string;
 }
 
-export const Checkbox: React.FC<PropTypes> = (props) => {
-  const {
-    disabled,
-    id,
-    text,
-    onChange,
-    checked,
-    value,
-  } = props;
+export const Checkbox: React.FC<PropTypes> = memo(
+  (props) => {
+    const {
+      disabled,
+      id,
+      text,
+      onChange,
+      checked,
+      value,
+    } = props;
```

Comment:
It's better to do destructuring in arguments instead of (props) - {disabled, id, text ...}

Reply:
thanks, I'll keep that in mind in future